src/infinistore.cpp

```diff
@@ -44,6 +44,7 @@ MM *mm;
 int gidx = 0;
 int lid = -1;
 uint8_t ib_port = -1;
+ibv_mtu active_mtu = IBV_MTU_1024;
```

Comment:
If you do not have comprehensive  solution for MTU, please revert MTU related code.  this MTU changes did nothing

Reply:
? from title of https://github.com/bd-iaas-us/infiniStore/issues/67, we want to use the MTU from device attribute, right?